Mathlib/NumberTheory/Sylvester.lean

```diff
@@ -0,0 +1,97 @@
+/-
+Copyright (c) 2025 Walter Moreira, Joe Stubbs. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Walter Moreira, Joe Stubbs
+-/
+import Init.Data.List.Nat.Pairwise
+import Mathlib.Algebra.Order.Star.Basic
+import Mathlib.Analysis.Normed.Field.Lemmas
+
+/-!
+# Sylvester sequence
+
+This file introduces the Sylvester's sequence.
+This is sequence [A000058](https://oeis.org/A000058) in [oeis].
+
+## Implementation notes
+
+We follow the presentantion from [Wikipedia](https://en.wikipedia.org/wiki/Sylvester%27s_sequence).
+
+## Main results
+
+- Basic facts.
+- Recurrence formula.
+- Sylvester sequence is strictly monotonic.
+- Pairwise co-primality.
```

Comment:
Could you include the names of the main results? (e.g., see `NumberTheory/Fermat` for the format)

Reply:
Thank you. `NumberTheory/Fermat` is a great example to follow.